src/styles/blocks/button.scss

```diff
@@ -0,0 +1,42 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+
+  text-decoration: none;
+
+  border: 2px solid $colorBlack;
+  border-radius: 40px;
+
+  @include fontStyle (700, 16px, 27px);
+  color: $colorWhite;
+  background: $colorBlack;
+
+  cursor: pointer;
+
+  transition: all 0.3s;
```

Comment:
Would be nice to use a variable for transition time, it's the same throughout the project.